Spanner/src/Connection/Grpc.php

```diff
@@ -152,6 +160,15 @@ public function __construct(array $config = [])
                 ? $config['authHttpHandler']
                 : null
         );
+
+        if (isset($config['enable-gcp'])) {
```

Comment:
Nit:

The name `enable-gcp` is ambiguous. It sounds like otherwise you can not use this on gcp, but it's actually an optimizer, right?

How about `enable-gcp-optimizer` or sth?